tensorboard/webapp/app_routing/effects/app_routing_effects_test.ts

```diff
@@ -96,7 +100,7 @@ describe('app_routing_effects', () => {
         },
         {
           routeKind: RouteKind.COMPARE_EXPERIMENT,
-          path: '/compare',
+          path: '/compare/:experimentIds',
```

Comment:
Because getRouteId is not called in app_routing_effects, we need to ensure the route ids it is expecting are well-formed. This has a bit of a big blast radius in this test because we now have to generate proper compare route ids.